Formula/vice.rb

```diff
@@ -14,59 +13,65 @@ class Vice < Formula
     sha256 "ab4044f958907bd7d756575fc97e0e42ffc24307c621176da0d0522feadb22f4" => :yosemite
   end
 
+  head do
```

Comment:
Inline this as `head "https://svn.code.sf.net/p/vice-emu/code/trunk/vice"` below the `sha256` line.